ubuntudesign/maas.io
# Use documentation builder

Now that [maas-docs](https://github.com/CanonicalLtd/maas-docs/) is in the new format, switch over to using `documentation-builder` to build the documentation.

- Change template to new format
- Merge include with template
- Change `make docs` to run `documentation-builder` with required options
- Document how to build docs in the README
- Set up redirects to redirect from `/docs/*` to the new `/docs/en/*`

QA
---

``` bash
pip3 install --user ubuntudesign.documentation-builder
make docs
make setup
make develop
```

Go to <http://127.0.0.1:8000/docs> - see it redirect to `/docs/en/`, and see that the docs work.
